resources/lang/de_DE/default.po

```diff
@@ -60,7 +60,7 @@ msgstr "No sleep found"
 
 #: resources/views/errors/405.twig:3 resources/views/errors/405.twig:5
 msgid "405: Method not allowed"
-msgstr ""
+msgstr "405: Method not allowed"
```

Comment:
Why not in german?

Reply:
"Method not allowed" is the official message of the status code. German Wikipedia also uses it: https://de.wikipedia.org/wiki/HTTP-Statuscode